crates/topos-tce-broadcast/src/lib.rs

```diff
@@ -3,22 +3,20 @@
 //! Abstracted from actual transport implementation.
 //! Abstracted from actual storage implementation.
 
+use double_echo::DoubleEcho;
+use ethers::types::Signature;
```

Comment:
re-export the `Signature` from `topos-crypto` to avoid scope deps to `topos-crypto` only